IliasDownloaderUniMA/__init__.py

```diff
@@ -163,21 +247,28 @@ def _determineItemType(self, url):
 
 	def _parseFileProperties(self, bs_item):
 		p = [i for i in bs_item.find_all('span', 'il_ItemProperty') if len(i.text) > 0]
+
 		if len(p[0].text.split()) > 1:
 			file_ending = ""
 		else:
 			file_ending = "." + p[0].get_text().split()[0]
-		file_size_tmp = p[1].get_text().replace(".","").replace(",", ".").split()
-		file_size = float(file_size_tmp[0])
-		if file_size_tmp[1] == "KB":
-			file_size *= 1e-3
-		elif file_size_tmp[1].lower() == "bytes":
-			file_size *= 1e-6
+
+		if len(p) > 1:
+			file_size_tmp = p[1].get_text().replace(".","").replace(",", ".").split()
+			file_size = float(file_size_tmp[0])
+			if file_size_tmp[1] == "KB":
+				file_size *= 1e-3
+			elif file_size_tmp[1].lower() == "bytes":
+				file_size *= 1e-6
+		else:
+			file_size = math.nan
+
 		p = [i for i in p if "Version" not in i.text]
 		if len(p) > 2:
 			file_mod_date = parsedate(self.translate_date(p[2].get_text()), dayfirst=True)
 		else:
 			file_mod_date = datetime.fromisoformat('2000-01-01')
+
 		return file_ending, file_size, file_mod_date
```

Comment:
Never been really proud about this method, it's by far the most ugliest one inside the whole class. 😅 So it seems to be a good idea to add a few comments regarding Ilias' quirks and refactor this one a bit in the near future. 

Are you willed to help?

Reply:
I'd be more than willing to help, but I think I'll two to three weeks before that.
I still have to send three PRs for Hacktoberfest and got a lot of other stuff to do first.

Reply:
The function was probably a prototype to get things off the ground by you?
I see nothing wrong with testing things things first and redoing them at some later point :shrug:

Reply:
If I try to contribute tests to this project at some point I'd think this would be a good method to start.

Reply:
> I'd be more than willing to help, but I think I'll two to three weeks before that.

Don't worry, that's not urgent.

> The function was probably a prototype to get things off the ground by you?

Exactly, even though I don't see a more elegant way to do it atm. 🤔

> If I try to contribute tests to this project at some point I'd think this would be a good method to start.

👍